handler/src/test/java/io/netty/handler/ssl/OpenSslTestUtils.java

```diff
@@ -24,4 +24,8 @@ private OpenSslTestUtils() {
     static void checkShouldUseKeyManagerFactory() {
         assumeTrue(OpenSsl.supportsKeyManagerFactory() && OpenSsl.useKeyManagerFactory());
     }
+
+    static boolean isBoringSSL() {
+        return "BoringSSL".equals(OpenSsl.versionString());
```

Comment:
Curious if we should be case-insensitive here.

Reply:
I think its fine ...